en/12_Project_Governance/09_Repository_Management.md

```diff
@@ -12,26 +12,26 @@ This document outlines the rules and processes for managing the GitHub repositor
 
 ### Naming convention
 
-Branches with a single integer (e.g. `5`) represent either the next *major* release (if there have been no tagged releases for that release line yet) or the next *minor* release.
+Branches with a single integer (e.g. `1`) represent either the next *major* release (if there have been no tagged releases for that release line yet) or the next *minor* release.
 
-Branches with an integer, a dot, and another integer (e.g. `5.2`) represent either the *next patch release* for the current minor, or a patch release for a previous minor version.
+Branches with an integer, a dot, and another integer (e.g. `1.2`) represent either the *next patch release* for the current minor, or a patch release for a previous minor version.
```

Comment:
Use more generic `1.2` instead of CMS numbers so we don't have to update it next time, since this page doesn't reference specific modules